setup.py

```diff
@@ -37,7 +41,7 @@ def read_relative_file(filename):
     long_description=read_relative_file('README.rst'),
     author='Yohann Gabory',
     author_email='yohann.gabory@novapost.fr',
```

Comment:
may be we can put here another contact email ?
novafloss@people-doc.com for example